.github/workflows/biome-check.yaml

```
@@ -64,24 +64,28 @@ jobs:
+          errors=$(echo "$clean_output" | grep "Found [0-9]* errors" | sed 's/Found \([0-9]*\) errors.*/\1/' || echo "0")
+          warnings=$(echo "$clean_output" | grep "Found [0-9]* warnings" | sed 's/Found \([0-9]*\) warnings.*/\1/' || echo "0")
+          # Use the found errors + warnings as total issues
+          total_issues=$((errors + warnings))
+
+          if [ "$individual_issues" -gt "$total_issues" ]; then
+            final_issues_count=$individual_issues
           else
-            total_issues=$found_total
+            final_issues_count=$total_issues
           fi
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)